packages/neutrino/karma.js

```diff
@@ -0,0 +1,68 @@
+const babelMerge = require('babel-merge');
+const merge = require('deepmerge');
+const omit = require('lodash.omit');
+const { join } = require('path');
+
+module.exports = adapter => config => {
+  const { neutrino } = adapter;
+
+  adapter
+    .command('karma')
+    .init();
+
+  if (neutrino.config.module.rules.has('compile')) {
```

Comment:
It feels like these really belong in the preset rather than in the Neutrino package. Could we instead maybe do something like either:
* make these files be mostly a shell that `require()`s the relevant parts from the preset
* have the presets dynamically register new commands, so we don't need to hardcode each test runner in the neutrino package?

Reply:
Hmm, I hadn't thought of it this way before, and it would take re-working the adapter pattern, but let me see what I can do.